vars/runSctTest.groovy

```diff
@@ -27,6 +27,14 @@ def call(Map params, String region, functional_test = false, Map pipelineParams
     export SCT_CONFIG_FILES=${test_config}
     export SCT_COLLECT_LOGS=false
 
+    if [[ -n "${params.stress_duration ? params.stress_duration : ''}" ]] ; then
```

Comment:
```suggestion
    if [[ -n "${params.stress_duration ? params.stress_duration : '0'}" ]] ; then
```

Reply:
but this change will make `if (params.stress_duration == "") {` from `vars/getJobTimeouts.groovy` to go to else condition which is wrong.

Reply:
it won't because the getJobTimeout is running earlier and will not affect on runSctTest, but there is no need to set this to 0, because if stress_parameter is not set in pipeline parameter, we will not export appropriate SCT env, and will use default value which is 0 and means that old behavior will be used (test_duration and stress command duration defined inside command)

Reply:
this is fine and follow the logic how we use env variables across the pipeline